food-truck.eve.md

```diff
@@ -156,148 +156,104 @@ commit
-- Draw accept button
-- Clicking applies entered credentials and reverts to truck settings screen
-
-## Cashier
-
-### Order Screen
-- Draw top banner
-- Draw truck name
-- Draw open/closed icon
-- Clicking brings up open/closed overlay
-- Clicking open sets truck status to open for business and reverts back to order screen
-- Clicking closed sets truck status to closed and reverts back to order screen
-- Draw menu items
-- Draw photo
-- Draw item name
-- Swiping right adds one to the cart
-- Swiping left removes one from the cart
-- Draw pay button
-- Clicking pay button takes you to the Stripe payment page/overlay
+```
+search
+  order = [#order #my-order number items status]
+  (ahead, message, my-status) = if status:"ready" then ("Your order is ready!","","ready")
+    else if count[given: [#order status:"pending" number < order.number]] = 1 then ("1","order ahead of you!","ahead")
+    else if ahead = count[given: [#order status:"pending" number < order.number]] then (ahead,"orders ahead of you!","ahead")
+    else ("0","orders ahead of you!","ahead")
+bind @browser
+  [#div class:"my-order" children:
+    [#div class:"order-confirmed" text:"Order confirmed!"]
+    [#div class:my-status text:ahead]
+    [#div class:"msg" text:message]
+    [#div class:"my-order-number" text:"Order #{{number}}"]
+    [#div class:"my-food" text:"{{count[given: items, per: (order, items.item)]}}x {{items.item.name}}"]
+    [#div class:"spacer"]
+  [#div class:"back" text:"❮ Back to Mama Rob's"]]
+```
 
-### Stripe Page
-- Draw email field
-- Draw CC number field
-- Draw MM/YY field
-- Draw CVV field
-- Draw “Confirm Order!” button
-- Confirming order processes payment
-- If payment succeeds, add the order from user cart into pending orders, clear the cart, and return to order screen
-- If payment fails, display error and stay on Stripe page
+```css
+.my-order {
+  border: 1px solid #000000;
+  border-radius: 6px;
+  flex: 0 0 600px;
+  margin-bottom: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+  
+.order-confirmed {
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 14px;
+  text-align: left;
+  flex: 0 0 50px;
+  padding-left: 20px;
+  padding-top: 20px;
+}
+
+.ahead {
+  flex: 1;
+  font-weight: bold;
+  font-size: 60px;
+  line-height: 60px;
+  text-align: center;
+  color: #b4b4b4;
+  flex: 0 0 100px;
+  padding-top: 40px;
+}
+
+.ready {
+  flex: 1;
+  font-weight: bold;
+  font-size: 36px;
+  line-height: 36px;
+  text-align: center;
+  color: green;
+  flex: 0 0 100px;
+  padding-top: 56px;
+}
+
+.msg {
+  flex: 1;
+  text-align: center;
+  flex: 0 0 40px;
+}
+
+.my-order-number {
+  text-decoration: underline;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 14px;
+  text-align: left;
+  height: 50px;
+  padding-left: 20px;
+  padding-top: 20px;
+}
+.my-food {
+  padding-left: 20px;
+}
+
+.spacer {
+  flex-grow: 10;
+}
+
+.back {
```

Comment:
We should be a little more disciplined about our classnames here, since anything else in the app that wants to use a name like `.back` will end up picking these styles up. In general it's good practice to have all of your selectors have two parts -- the classname of the container (in this case, maybe the page) and the classname of the thing to style (in this case, the back button). E.g. `.orders-page .back-btn { ...`